Dockerfile

```
@@ -2,162 +1,91 @@
-        librsvg2-dev \
-        fonts-liberation \
-        fonts-dejavu \
-        fontconfig && \
-    # Install Node.js
-    curl -fsSL https://deb.nodesource.com/setup_18.x | bash - && \
-    apt-get install -y nodejs && \
-    # Clean up
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+# Only runtime deps
+RUN apk add --no-cache openssl ncurses-libs
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)